ts/session/sending/MessageQueue.ts

```diff
@@ -87,18 +90,20 @@ export class MessageQueue {
    */
   public async sendToGroup(
     message: ClosedGroupMessageType,
-    sentCb?: (message: RawMessage) => Promise<void>
+    sentCb?: (message: RawMessage) => Promise<void>,
+    groupPubKey?: PubKey
   ): Promise<void> {
-    let groupId: PubKey | undefined;
+    let destinationPubKey: PubKey | undefined = groupPubKey;
     if (message instanceof ExpirationTimerUpdateMessage || message instanceof ClosedGroupMessage) {
-      groupId = message.groupId;
+      destinationPubKey = groupPubKey ? groupPubKey : message.groupId;
     }
 
-    if (!groupId) {
+    if (!destinationPubKey) {
       throw new Error('Invalid group message passed in sendToGroup.');
     }
+
     // if groupId is set here, it means it's for a medium group. So send it as it
-    return this.sendToPubKey(PubKey.cast(groupId), message, sentCb);
+    return this.sendToPubKey(PubKey.cast(destinationPubKey), message, sentCb, true);
```

Comment:
note to myself (do not mark as resolved)

find a way to not have to pass the groupPubKey nor the `sentCb, true);` here